examples/client/client.py

```
@@ -15,62 +21,133 @@
+    This method automatically: 
+    - Reads HEDERA_NETWORK (defaults to "testnet")
+    - Reads OPERATOR_ID (required)
+    - Reads OPERATOR_KEY (required)
+    - Creates and configures the client
+    """
+    print("\n--- Method 2: Client.from_env() (Recommended) ---")
+
+    # One line to create a fully configured client! 
+    client = Client.from_env()
+
+    print(f"✅ Client created with single line!")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)